docs/source/usage.rst

```diff
@@ -11,12 +11,22 @@ You can find a full list of variables in `env.example`:
 
 .. literalinclude:: ../../env.example
 
-You can set these variables in the command line:
+You can set then load all of these variables at once in the command line:
+
+.. code-block:: bash
+
+    set -o allexport
+    source .env
+    set +o allexport
+
+Or you can also set individual variables in the command line:
```

Comment:
Is this suggesting that people directly edit `env.example`? I'm confused about what this command will do exactly

Reply:
In the env.example file the initial instructions say the following.

> # This is a sample .env file.
> # Duplicate this file as .env in the root of the project
> # and update the environment variables to match your
> # desired config.

There are no instructions to actually load this through and the pipeline currently doesn't load this automatically either.

Reply:
Ah got it!